gcs/debug_bucket.go

```diff
@@ -43,6 +43,13 @@ type debugBucket struct {
 	nextRequestID uint64
 }
 
+func GetLoggerAndWrappedWithDebugBucket(b Bucket, logger *log.Logger) *debugBucket {
```

Comment:
Make the newDebugBucket method as public instead of seperate method.

Reply:
fixed